### PR TITLE
Concierge Posts are added to empty discussions and Delphis accepts responses from App

### PR DESCRIPTION
**Pull request by tim-harper**

### CODE REVIEW

File: graph/resolver/schema.resolvers.go [hunk 1]
```diff
@@ -409,6 +409,21 @@ func (r *mutationResolver) DeleteDiscussionTags(ctx context.Context, discussionI
...
 }
 
+func (r *mutationResolver) ConciergeMutation(ctx context.Context, discussionID string, mutationID string, selectedOptions []string) (bool, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return false, fmt.Errorf("Need auth")
+	}
+
+	// Only allow the mod to delete discussion tags
+	modCheck, err := r.DAOManager.CheckIfModeratorForDiscussion(ctx, authedUser.UserID, discussionID)
```
[the remark this answers is not in the document]
Comment (tim-harper) [reply graph/resolver/schema.resolvers.go:419]: I agree. passing in authedUser.UserID now

File: internal/backend/concierge_post.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+	post := model.Post{
+		ID:            util.UUIDv4(),
+		CreatedAt:     time.Now(),
+		UpdatedAt:     time.Now(),
+		DiscussionID:  &discussionID,
+		ParticipantID: &participantID,
+		PostContent: &model.PostContent{
+			Content: "You have some nice flair! If you like, you can set it up so people from the same groups can join automatically." +
+				"Otherwise, they'll need the VIP link or you'll have to approve them before they can join the chat.",
```
[the remark this answers is not in the document]
Comment (tim-harper) [reply internal/backend/concierge_post.go:100]: :eagle_eye:

File: graph/types/schema.graphqls [hunk 3]
```diff
@@ -92,6 +92,8 @@ type Mutation {
 
   addDiscussionTags(discussionID: ID!, tags:[String!]): [Tag!]
   deleteDiscussionTags(discussionID: ID!, tags:[String!]): [Tag!]
+
+  conciergeMutation(discussionID: ID!, mutationID: ID!, selectedOptions:[String!]): Boolean! # Heterogeneous endpoint. Think about what we want to return here. Could use entities? What does that gain us?
```
[the remark this answers is not in the document]
Comment (tim-harper) [reply graph/types/schema.graphqls:96]: I think that makes sense. It definitely is interacted with by a Post but are we going to re-show the post to the user? I can modify this once the invitation PR is merged

[the remark this answers is not in the document]
Comment (nedrocks) [reply graph/types/schema.graphqls:96]: I think so. We will definitely show the updated post state to the user

[the remark this answers is not in the document]
Comment (tim-harper) [reply graph/types/schema.graphqls]: updated

**Pull request merged**